pynvjitlink/api.py

```diff
@@ -16,6 +16,10 @@ class InputType(Enum):
     LIBRARY = 6
 
 
+def nvjitlink_version():
```

Comment:
I am wondering if this should be a (perhaps class) method instead? Such as `NvJitLinker.version()`?

Reply:
I think that seems ambiguous or confusing, as it's providing the version of the underlying nvJitLink library, and `NvJitLinker.version` might appear to be the version of the pynvjitlink library.

Reply:
I agree with Graham, it feels better as a free function named like “nvjitlink” and not “NvJitLinker.”